services/grpc/lightning.subscriptions.js

```diff
@@ -83,6 +84,36 @@ function subscribeTransactions() {
   })
   return call
 }
+/**
+ * Call lnd grpc subscribeTransactions method and emit events on updates to the stream
+ * @return {Call} Grpc Call
+ */
+function subscribeChannelBackups() {
+  if (this.service.subscribeChannelBackups) {
+    grpcLog.info('subscribeChannelBackups')
```

Comment:
This log line can go

Reply:
Reached out to that line and asked to GTFO